### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by August66** [chenchongthu/DeepCoNN#19]
Hello i want to ask if your results are overfitting on the test set,but the train loss has been decreasing.
